webpack/private/webpack_bundle.bzl

```diff
@@ -196,10 +197,20 @@ def _impl(ctx):
     executable = ctx.executable.webpack_exec_cfg
     execution_requirements = {}
 
+    no_copy_bin_inputs = []
     if ctx.attr.supports_workers:
         executable = ctx.executable.webpack_worker_exec_cfg
         execution_requirements["supports-workers"] = str(int(ctx.attr.supports_workers))
 
+        no_copy_bin_inputs.append(ctx.file._worker_js)
+
+        # TODO: get this path right!
```

Comment:
Is this TODO still something to solve?

Reply:
Yes. I'll clean that up before landing. The logic just needs to be generalized here and have to make sure there are test cases for both code paths.

Reply:
✅